js/bibox.js

```diff
@@ -176,7 +176,7 @@ module.exports = class bibox extends Exchange {
                         'max': undefined,
                     },
                     'price': {
-                        'min': undefined,
+                        'min': Math.pow (10, -precision['price']),
```

Comment:
The reason for not setting it, is that the min limit may be way above precision, and if we serve a lower limit value based on precisions – that may be incorrect. Say, a precision of 8 and a min price of 0.12345678.

Are you sure that with bibox min price is always based on price precision?

Reply:
No, but we assume the same for the amount. Either we should set both or remove both.